actionpack/lib/action_controller/metal/strong_parameters.rb

```diff
@@ -580,6 +580,22 @@ def stringify_keys # :nodoc:
       dup
     end
 
+    def method_missing(method_sym, *args, &block)
+      if @parameters.respond_to?(method_sym)
+        message = <<-DEPRECATE.strip_heredoc
+          Method #{ method_sym } is deprecated and will be removed in Rails 5.1. StrongParameters no longer inherit from
```

Comment:
I feel like this could be revised a bit.

```
Method #{ method_sym } is deprecated and will be removed in Rails 5.1, as `ActionController::Parameters` no longer ...
```

Strong Parameters is like a nickname for this feature, but user is seeing `ActionController::Parameters`.